naga-cli/src/bin/naga.rs

```diff
@@ -236,27 +242,16 @@ impl std::error::Error for CliError {}
 fn run() -> Result<(), Box<dyn std::error::Error>> {
     env_logger::init();
 
-    // Initialize default parameters
-    //TODO: read the parameters from RON?
-    let mut params = Parameters::default();
-
     // Parse commandline arguments
     let args: Args = argh::from_env();
     if args.version {
         println!("{}", env!("CARGO_PKG_VERSION"));
         return Ok(());
     }
-    let (input_path, input) = if let Some(path) = args.files.first() {
-        let path = Path::new(path);
-        (path, fs::read(path)?)
-    } else if let Some(path) = &args.stdin_file_path {
-        let mut input = vec![];
-        std::io::stdin().lock().read_to_end(&mut input)?;
-        (Path::new(path), input)
-    } else {
-        return Err(CliError("Input file path is not specified").into());
-    };
-    let output_paths = args.files.get(1..).unwrap_or(&[]);
+
+    // Initialize default parameters
+    //TODO: read the parameters from RON?
+    let mut params = Parameters::default();
```

Comment:
nitpick(non-blocking): This move of init. for the `params` binding doesn't seem necessary? I wouldn't change it now, since both author and reviewer now understand the code, but it tooks me a bit to figure out that this movement doesn't meaningfully change anything.